bears/general/FilenameBear.py

```diff
@@ -38,6 +46,21 @@ def run(self, filename, file,
         """
         head, tail = os.path.split(filename)
         filename_without_extension, extension = os.path.splitext(tail)
+
+        if file_naming_convention is None:
+            self.warn('Please specify a file naming convention explicitly. '
+                      'Using the default "snake" naming convention...')
+            file_naming_convention = 'snake'
```

Comment:
The first point [here](https://github.com/coala/coala-bears/issues/1083#issuecomment-288518740) states that we use `snake` if the user doesn't mention explicitly. Is that okay ? @Makman2